### ISSUE TITLE
import/export | Example in README did not work

### ISSUE THREAD

**Issue by CharJon** [CharJon/GeCO#85]
The example code in the README did not work:
```python
from geco import knapsack
knapsack.yang_instance(n=5, seed=1)
```

For this we would have to change our __init__.py files to the same style [networkx is using](https://github.com/networkx/networkx/blob/master/networkx/generators/__init__.py).

Shall we?

**Comment by mmghannam** [comment]
> The example code in the README did not work:
> 
> ```python
> from geco import knapsack
> knapsack.yang_instance(n=5, seed=1)
> ```
> 
> For this we would have to change our **init**.py files to the same style [networkx is using](https://github.com/networkx/networkx/blob/master/networkx/generators/__init__.py?rgh-link-date=2021-01-21T11%3A01%3A06Z).
> 
> Shall we?

Yeah I think it would be better yes, and there shouldn't be any conflicts since we are using the first author name in the function names.

**Comment by mmghannam** [comment]
closed by #86
